examples/jenkins/README.md

```diff
@@ -35,7 +35,7 @@ Steps
 
     1. Go back to your browser, refresh and select the rubyJob build job.
     2. Choose `Configure`.
-    3. Locate the KUBECONFIG_CREDENTIALS parameter, and replace the default value with the contents of the `openshift.local.config/master/openshift-client.kubeconfig` file.
+    3. Locate the KUBECONFIG_CREDENTIALS parameter, and replace the default value with the contents of the `openshift.local.config/master/openshift-master.kubeconfig` file.
```

Comment:
I plan to update the Jenkins example to use a service account so this step isn't necessary